clientd3d/client.rc

```diff
@@ -18,6 +18,754 @@
 /////////////////////////////////////////////////////////////////////////////
 #undef APSTUDIO_READONLY_SYMBOLS
 
+/////////////////////////////////////////////////////////////////////////////
+// Brazilian resources
+
+#if !defined(AFX_RESOURCE_DLL) || defined(AFX_TARG_DEU)
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+#pragma code_page(1252)
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Menu
+//
+
+IDR_MAINMENU MENU
+BEGIN
+    POPUP "&Jogo"
+    BEGIN
+        MENUITEM "&Conectar-se",                  3061
+        MENUITEM "&Desconectar",                   3062
+        MENUITEM "&Prefer�ncias",           3060
+        MENUITEM "Configura��es",               186
+        MENUITEM "Alterar senha",            3506
+        MENUITEM SEPARATOR
+        MENUITEM "&Imprimir mapa",               3507
+        MENUITEM SEPARATOR
+        MENUITEM "&Sair do jogo",            3002
+    END
+    POPUP "&Op��es"
+    BEGIN
+        MENUITEM "&M�sica",                      3034
+        MENUITEM "&Sons",                     3033
+        MENUITEM SEPARATOR
+        POPUP "&Cores"
+        BEGIN
+            POPUP "Janela principal"
+            BEGIN
+                MENUITEM "&Janela do jogo",              3024
+                MENUITEM "&Feiti�os selecionados",            3030
+                MENUITEM "&N�meros no invent�rio",         3050
+                MENUITEM "&Propriedades",              3047
+                MENUITEM "Mensagens do sistema",         3504
+                MENUITEM "&Campo de mensagens",                   3032
+            END
+            POPUP "Sele��o de itens"
+            BEGIN
+                MENUITEM "Itens n�o selecionados",       3042
+                MENUITEM "Itens selecionados",     3027
+            END
+            POPUP "Gr�ficos de barra"
+            BEGIN
+                MENUITEM "&Barras normais",              3500
+                MENUITEM "&Barras completas",             3501

```

Comment:
We can't have the same hotkey (B, in this case) for multiple items in the same menu.  I'm afraid this is going to require a thorough audit.

Reply:
I've gone through and updated hotkeys by comparing available hotkeys to the English version and ensuring that no duplicate hotkeys exist within the same level of each menu.